src/test/unit/test_auto_instrument_handler.py

```diff
@@ -32,33 +32,67 @@ def test_hierarchy_happy_flow(monkeypatch):
 def test_import_error(monkeypatch):
     monkeypatch.setenv(ORIGINAL_HANDLER_KEY, "blabla.not.exists")
 
-    with pytest.raises(ImportError):
```

Comment:
Why not do it with `pytest.raises`?
Same issue regarding the other tests in this file as well.

Reply:
That's a great question!
I did use pytest.raises in the first place, but then I noticed that it removes the context of the message, and I want to validate that there was no context (to follow AWS behavior!) :D

Reply:
Add a comment explaining why don't we use it